.github/workflows/tasklist-ci-test-reusable.yml

```diff
@@ -130,7 +130,7 @@ jobs:
 
       - name: Upload failed test results
         uses: ./.github/actions/collect-test-artifacts
-        if: failure()
+        if: failure() || cancelled()
```

Comment:
:x: Please open a PR to main with just this change.

Reply:
https://github.com/camunda/camunda/pull/37114